Reference/Searching/Examine/Indexing/index.md

```diff
@@ -164,24 +164,17 @@ To create this index we need five things:
 using Examine.Lucene;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
+using Umbraco.Cms.Core.Hosting;
 using Umbraco.Cms.Core.Services;
 using Umbraco.Cms.Infrastructure.Examine;
-using IHostingEnvironment = Umbraco.Cms.Core.Hosting.IHostingEnvironment;
 
-namespace MySite.MyCustomIndex
+namespace Umbraco.Docs.Samples.Web.CustomIndexing
 {
     public class ProductIndex : UmbracoExamineIndex
     {
         public ProductIndex(
-            ILoggerFactory loggerFactory,
-            string name,
-            IOptionsMonitor<LuceneDirectoryIndexOptions> indexOptions,
-            IHostingEnvironment hostingEnvironment,
-            IRuntimeState runtimeState) : base(loggerFactory,
-            name,
-            indexOptions,
-            hostingEnvironment,
-            runtimeState)
+            ILoggerFactory loggerFactory, string name, IOptionsMonitor<LuceneDirectoryIndexOptions> indexOptions, IHostingEnvironment hostingEnvironment, IRuntimeState runtimeState)
```

Comment:
Isn't it a shame, that the "list overview" of the items here is lost? 🤔 
I think the "list overview" is easier to read - might just be me!